give/methods/give/server/order.js

```diff
@@ -51,12 +54,15 @@ function order(orderData, instant, id){
 
     orderData["ip-address"] = ip
 
+    // strongly force CVV on acctions that aren't a saved account
+    if (!orderData["customer-vault-id"]) {
+      orderData["cvv-reject"] = "P|N|S|U"
```

Comment:
this is a way to force CVV match on all counts. We can't turn this setting on in NMI because then saved accounts wouldn't work. `orderData["customer-vault-id"]` is present because `orderData.savedAccount` was not `false`